### PR TITLE
Anjali_Asgnmt8

### PR DESCRIPTION
**Pull request by AnjaliSoct2020**

### CODE REVIEW

File: src/anjaliS/DivisionByWhile.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	   void divisibleBy13or7(int start,int end){
+	    System.out.println("Divisible by 7 or 13, numbers are");
+	    while(start<=end)
+	   {
+	    if(start%7==0 )
+	     System.out.println(start +" is divisible by 7");
+	     start++;
+	    if(start%13==0)
+	     System.out.println(start +" is divisible by 13");
+	     start++;
```
Comment (harshvegada) [inline src/anjaliS/DivisionByWhile.java:41]: Incorrect code, increment always happens once only. did you compare your output with expected output? @AnjaliSoct2020

**Review by harshvegada — APPROVED** [review]
Code not one requirement , Please refer someone's code
Merging!

File: src/anjaliS/DivisionByWhile.java (hunk 1 above)
Comment (AnjaliSoct2020) [reply src/anjaliS/DivisionByWhile.java:41]: yes checked but i think got missed, corrected now ,shall i push again code correct one??

Comment (AnjaliSoct2020) [reply src/anjaliS/DivisionByWhile.java:41]: yes checked but i think got missed, corrected now ,shall i push again code correct one??@harshvegada

**Pull request merged**